integration-test/src/main/java/com/sequenceiq/it/cloudbreak/filesystem/FilesystemUtil.java

```diff
@@ -0,0 +1,129 @@
+package com.sequenceiq.it.cloudbreak.filesystem;
+
+import java.io.ByteArrayInputStream;
+import java.security.PrivateKey;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.testng.Assert;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.Base64;
+import com.google.api.client.util.SecurityUtils;
+import com.google.api.services.storage.Storage;
+import com.google.api.services.storage.model.Objects;
+import com.google.api.services.storage.model.StorageObject;
+import com.microsoft.azure.datalake.store.ADLStoreClient;
+import com.microsoft.azure.datalake.store.oauth2.AccessTokenProvider;
+import com.microsoft.azure.datalake.store.oauth2.ClientCredsTokenProvider;
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.sequenceiq.it.util.ResourceUtil;
+
+
+
+public class FilesystemUtil {
+
+    private static final String AZURE_TOKEN_URL = "https://login.microsoftonline.com/";
+
+    private static final String STORAGE_SCOPE = "https://www.googleapis.com/auth/devstorage.read_write";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FilesystemUtil.class);
+
+    private FilesystemUtil() {
+    }
+
+    protected static void cleanUpFiles(ApplicationContext applicationContext, Map<String, String> cloudProviderParams,
+            String fsType, String fsName, String folderPrefix, String containerName) throws Exception {
+        switch (fsType) {
+            case "ADLS":
+                deleteAdlsObjects(cloudProviderParams, fsName, folderPrefix);
+                break;
+            case "GCS":
+                deleteGcsObjects(applicationContext, cloudProviderParams, fsName, folderPrefix);
+                break;
+            case "WASB":
+                deleteWasbContainer(cloudProviderParams, fsName, containerName);
```

Comment:
This one is catching all exception and handling them, others are delegating error handling to caller. Would be nice a common solution